core/whistle_apps-1.0.0/src/whapps_maintenance.erl

```diff
@@ -949,3 +951,27 @@ show_status(CallId, 'true', Resp) ->
     lager:info("Media Server: ~s", [wh_json:get_value(<<"Switch-Hostname">>, Resp)]),
     lager:info("Responding App: ~s", [wh_json:get_value(<<"App-Name">>, Resp)]),
     lager:info("Responding Node: ~s", [wh_json:get_value(<<"Node">>, Resp)]).
+
+
+-spec delete_system_media_references() -> 'no_return'.
+delete_system_media_references() ->
+    Db = <<"media">>,
+    {'ok', MediaDoc} = couch_mgr:open_cache_doc(?WH_CONFIG_DB, Db),
+    TheKey = <<"default">>,
+    MediaDefault = wh_json:get_value(TheKey, MediaDoc),
+    Deleter =
+        fun (Key, Value, Acc) ->
+                case starts_with_system_media(Key) orelse
```

Comment:
We dropline `orelse`, `andalso`, etc, AND we dropline the `of` in multi-line case:

```erlang
case starts_with_system_media(Key)
        orelse starts_with_system_media(Value)
of
    'true' -> Acc;
    'false' -> [{Key, Value} | Acc]
end
```
Also, I don't think you need to check Key in this case.